testsuite/integration/basic/pom.xml

```diff
@@ -1326,6 +1365,37 @@
                                 </configuration>
                             </execution>
 
+                            <!-- Tests against the install with batch-jberet -->
+                            <execution>
+                                <id>batch-jberet-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <jboss.install.dir>${basedir}/target/wildfly-batch</jboss.install.dir>
+                                        <jboss.home>${project.build.directory}/wildfly-batch</jboss.home>
+                                        <jboss.home.dir>${project.build.directory}/wildfly-batch</jboss.home.dir>
+                                        <jbossas.dist>${project.build.directory}/wildfly-batch</jbossas.dist>
+                                        <jboss.dist>${project.build.directory}/wildfly-batch</jboss.dist>
+                                        <!-- Override the standard module path that points at the shared module set from dist -->
+                                        <module.path>${project.build.directory}/wildfly-batch/modules${path.separator}${basedir}/target/modules</module.path>
+                                    </systemPropertyVariables>
+                                    <environmentVariables>
```

Comment:
system property `jboss.inst` is also used by some of [these tests](https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/batch/deployment/JobControlTestCase.java#L85), we may need to pass it too. But looks like these tests passed without it, probably because we ran them without security manager?

Reply:
Thanks for reviewing Cheng. Yes, they it fails with the security manager. 

@bstansberry what is the position against whether we should run the layers tests under security manager? currently we do not have a CI with this configuration

Reply:
That would require an entire separate CI job. I don't think it's worth it to add a per-PR job for that. Maybe a nightly job, but I'd be concerned it makes the set of controls of what's enabled overly complex. (Probably they already are!)  Something to look at in the future though.

Reply:
Ok, I leave the sec manager concerns for now.

>  I'd be concerned it makes the set of controls of what's enabled overly complex. (Probably they already are!)

Yes, I think they are already.

Reply:
Hello @chengfang Would you mind to approval the changes on this PR?